fedimint-cli/src/lib.rs

```diff
@@ -359,6 +364,35 @@ enum AdminCmd {
     GuardianConfigBackup,
 }
 
+#[derive(Debug, Clone, Subcommand)]
+enum DecodeType {
+    /// Decode an invite code string into a JSON representation
+    InviteCode { invite_code: InviteCode },
+    /// Decode a string of ecash notes into a JSON representation
+    Notes { notes: OOBNotes },
+}
+
+#[derive(Debug, Clone, Subcommand)]
+enum EncodeType {
+    /// Encode connection info from its constituent parts
+    InviteCode {
+        #[clap(long = "url")]
+        url: SafeUrl,
```

Comment:
I think if the field name is the same as the long argument name you can just annotate it as `#[clap(long)]`

Reply:
updated